singlestore/datadog_checks/singlestore/values_cleaner.py

```diff
@@ -0,0 +1,63 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import AnyStr, Callable, Optional, Sequence, Union
+
+from datadog_checks.singlestore.queries import AGGREGATORS, MV_GLOBAL_STATUS
+
+
+class SingleStoreCleaningException(Exception):
+    pass
+
+
+UNITS = {
+    'GB': 1e9,
```

Comment:
Are the units definitely case-sensitive?

Reply:
As far as testing goes, yes they are